internal/webhooks/vspheredeploymentzone_webhook_test.go

```diff
@@ -47,12 +50,13 @@ func TestVSphereDeploymentZone_Default(t *testing.T) {
 		// Need to reinit the test variable
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
-			vdz := VSphereDeploymentZone{
-				Spec: VSphereDeploymentZoneSpec{
+			vdz := infrav1.VSphereDeploymentZone{
+				Spec: infrav1.VSphereDeploymentZoneSpec{
 					ControlPlane: tt.boolPtr,
 				},
 			}
-			vdz.Default()
+			webhook := VSphereDeploymentZoneWebhook{}
+			g.Expect(webhook.Default(context.TODO(), &vdz)).NotTo(HaveOccurred())
```

Comment:
Let's use `context.Background()`.

We will never replace the context here (note: please check for also other places :-) .


```suggestion
			g.Expect(webhook.Default(context.Background(), &vdz)).NotTo(HaveOccurred())
```

Reply:
Fixed.